app/components/Products.jsx

```diff
@@ -2,30 +2,36 @@
 import React from 'react';
 import {Link} from 'react-router'
 
+import Paper from 'material-ui/Paper'
+
 export default function Products (props) {
 
+//change to products.all
```

Comment:
can't remember where we left this off.... The store's state seems to still have `products` key, instead of `all`... Not sure what happened here. It's possible I forgot to complete the changes to the reducer. There might be some issues associated with this as I remember Luisa changing something associated with this decision in one of the other files...